src/components/Common/ProfileSelector/ProfileSelector.tsx

```diff
@@ -53,14 +53,28 @@ export default function ProfileSelector(props: ProfileSelectorSpec): JSX.Element
 		<div>
 			<div className="bg-blue-darker-blue rounded-md p-4 pt-2">
 				<Title>{t('profileSelector.title')}</Title>
-				<div className="mt-4 grid grid-cols-3 grid-flow-col auto-cols-min">
-					<div className="flexflex-col justify-between">
+				<div className=" flex items-center">
+					<div className="flex flex-col justify-between">
 						<SelectButton<RubberColor> direction="left" name={t('profileSelector.eraser')} value={props.profile.avatar.rubberColor} list={rubberColors} setValue={(v) => setRubberColor(v)} />
 						<SelectButton<BodyColor> direction="left" name={t('profileSelector.color')} value={props.profile.avatar.bodyColor} list={bodyColors} setValue={(v) => setBodyColor(v)} />
 						<SelectButton<FaceType> direction="left" name={t('profileSelector.face')} value={props.profile.avatar.faceType} list={faceTypes} setValue={(v) => setFaceType(v)} />
 					</div>
 
-					<div className="flex items-center">
+					<div style={{ maxHeight: '180px', maxWidth: '180px' }} className="flex flex-grow mx-auto px-4 relative">
+						<div style={{ position: 'absolute', right: 20, bottom: 0, zIndex: 1 }}>
+							<button
+								onClick={randomizeAvatar}
+								style={{
+									border: 'none',
+									height: '40px',
+									width: '40px',
```

Comment:
Maybe use EM to adapt the size to mobile? I have no strong feelings on this one, px is fine by me, just suggesting.

Reply:
mah, not a fan of this em rem stuff. I tested on phone, it's looking fine